the *64 apis are useless as musl is always 64-bit for them.
even on glibc, passing -D_FILE_OFFSET_BITS=64 defines 64-bit variants for non-64 names.
this legacy api is removed in musl-1.2.4 unless -D_LARGEFILE64_SOURCE,
omit it from here to fix downstream use without having to pass defines.
--
diff --git a/mkspecs/common/posix/qplatformdefs.h b/mkspecs/common/posix/qplatformdefs.h
index ffae2ac3..665ae46f 100644
--- a/mkspecs/common/posix/qplatformdefs.h
+++ b/mkspecs/common/posix/qplatformdefs.h
@@ -14,28 +14,28 @@
 
 #if defined(QT_USE_XOPEN_LFS_EXTENSIONS) && defined(QT_LARGEFILE_SUPPORT)
 
-#define QT_STATBUF              struct stat64
-#define QT_FPOS_T               fpos64_t
-#define QT_OFF_T                off64_t
+#define QT_STATBUF              struct stat
+#define QT_FPOS_T               fpos_t
+#define QT_OFF_T                off_t
 
-#define QT_STAT                 ::stat64
-#define QT_LSTAT                ::lstat64
-#define QT_TRUNCATE             ::truncate64
+#define QT_STAT                 ::stat
+#define QT_LSTAT                ::lstat
+#define QT_TRUNCATE             ::truncate
 
 // File I/O
-#define QT_OPEN                 ::open64
-#define QT_LSEEK                ::lseek64
-#define QT_FSTAT                ::fstat64
-#define QT_FTRUNCATE            ::ftruncate64
+#define QT_OPEN                 ::open
+#define QT_LSEEK                ::lseek
+#define QT_FSTAT                ::fstat
+#define QT_FTRUNCATE            ::ftruncate
 
 // Standard C89
-#define QT_FOPEN                ::fopen64
-#define QT_FSEEK                ::fseeko64
-#define QT_FTELL                ::ftello64
-#define QT_FGETPOS              ::fgetpos64
-#define QT_FSETPOS              ::fsetpos64
+#define QT_FOPEN                ::fopen
+#define QT_FSEEK                ::fseeko
+#define QT_FTELL                ::ftello
+#define QT_FGETPOS              ::fgetpos
+#define QT_FSETPOS              ::fsetpos
 
-#define QT_MMAP                 ::mmap64
+#define QT_MMAP                 ::mmap
 
 #else // !defined(QT_USE_XOPEN_LFS_EXTENSIONS) || !defined(QT_LARGEFILE_SUPPORT)
 
